### PR TITLE
regenerate output for demographic

### PR DESCRIPTION
**Pull request by wangben718**
Difference found:
1. Decimal digits
2. Line 31: min -max for high dose was 13.7 - 34.5

### CODE REVIEW

File: output/tlf-demographic.out.pilot3 [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  Mean (sd)                           23.63 (3.67)        25.06 (4.27)           25.35 (4.16)    
+  Median                                  23.40               24.30                 24.80        
+  Min - Max                           15.00 - 33.30       17.70 - 40.10         13.70 - 34.60    
+MMSE Total                                                                                       
+  Mean (sd)                           18.05 (4.27)        17.87 (4.22)           18.51 (4.16)    
+  Median                                  19.50               18.00                 20.00        
+  Min - Max                           10.00 - 23.00       10.00 - 24.00         10.00 - 24.00    
+—————————————————————————————————————————————————————————————————————————————————————————————————
+
+Program: tlf_demographic.Rmd
```
Comment (laxamanaj) [inline output/tlf-demographic.out.pilot3:38]: Change footnote to say `Program: tlf_demographic.R` .

File: output/tlf-demographic.out.pilot3 [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  Min - Max                           15.00 - 33.30       17.70 - 40.10         13.70 - 34.60    
+MMSE Total                                                                                       
+  Mean (sd)                           18.05 (4.27)        17.87 (4.22)           18.51 (4.16)    
+  Median                                  19.50               18.00                 20.00        
+  Min - Max                           10.00 - 23.00       10.00 - 24.00         10.00 - 24.00    
+—————————————————————————————————————————————————————————————————————————————————————————————————
+
+Program: tlf_demographic.Rmd 
+2023-02-24 11:35:44
+
```
Comment (laxamanaj) [inline output/tlf-demographic.out.pilot3:40]: For the filename of the output, can you please update to `tlf-demographic-pilot3.out`?

**Review by laxamanaj — CHANGES_REQUESTED** [review]
Hi, @wangben718 

I added some minor comments.  Hope they can be updated soon.  Thanks.

Joel

**Comment by laxamanaj** [comment]
Hi, @wangben718 .  Thanks for pushing for review.  One question, did you also push the script which creates the output, `tlf-demographic.R`.  In this PR, when I click on files changed I only see the `tlf-demographic-pilot3.out` has changed, but I don't see the script updates.
image

**Comment by wangben718** [comment]
> Hi, @wangben718 . Thanks for pushing for review. One question, did you also push the script which creates the output, `tlf-demographic.R`. In this PR, when I click on files changed I only see the `tlf-demographic-pilot3.out` has changed, but I don't see the script updates. image

Hi @laxamanaj , I have updated the tlf-demographic.R. I got a little confused about the workflow. So now we are not using `.rmd` any more and we revised `.R` file directly when re-creating TLFs for submission. Is that correct?

File: output/tlf-demographic.out.pilot3 (hunk 2 above)
Comment (laxamanaj) [reply output/tlf-demographic.out.pilot3]: this is fixed.  Thanks, @wangben718 !

File: output/tlf-demographic.out.pilot3 (hunk 1 above)
Comment (laxamanaj) [reply output/tlf-demographic.out.pilot3]: This is fixed.  Thanks, @wangben718 .

**Review by laxamanaj — APPROVED** [review]
Thanks for the udpates, @wangben718 . I also wanted to note that I just updated the scripts so that the extension are now `.R` instead of `.r`.

**Comment by laxamanaj** [comment]
> > Hi, @wangben718 . Thanks for pushing for review. One question, did you also push the script which creates the output, `tlf-demographic.R`. In this PR, when I click on files changed I only see the `tlf-demographic-pilot3.out` has changed, but I don't see the script updates. image
> 
> Hi @laxamanaj , I have updated the tlf-demographic.R. I got a little confused about the workflow. So now we are not using `.rmd` any more and we revised `.R` file directly when re-creating TLFs for submission. Is that correct?

That's correct, @wangben718 .  The .R script and demog table output looks good now.  I will merge.

**Pull request merged**